README.md

```diff
@@ -435,20 +423,13 @@ If a custom annotation is required, the annotation should be documented in the p
 <a name="miscellaneous"/>
 ## Miscellaneous
 
-`and` is preferred over `&&`.
-
-`or` is preferred over `||`.
-
-`is` is preferred over `==`.
+`&&` is preferred over `and`.
 
-`not` is preferred over `!`.
+`||` is preferred over `or`.
```

Comment:
can note that the reason here is to minimize the mental gap between frontend code and ruby